docs/iris/src/whatsnew/1.7.rst

```diff
@@ -198,6 +198,17 @@ Bugs fixed in v1.7.3
 * Cubes that contain a pseudo-level coordinate can now be saved to PP.
 * Fixed a bug in the FieldsFile loader that prevented it always loading all available fields.
 
+Bugs fixed in v1.7.4
+^^^^^^^^^^^^^^^^^^^^
+* :meth:`Coord.guess_bounds` can now deal with circular coordinates.
+* :meth:`Coord.nearest_neighbour_index` can now work with descending bounds.
+* Passing `weights` to :meth:`Cube.rolling_window` no longer prevents other
+  other keyword arguments from being passed to the aggregator.
```

Comment:
I missed it on first reading ... maybe because it's split over two lines. Nice emoji btw!